test/ca_updatedb.c

```diff
@@ -27,6 +27,7 @@ int main(int argc, char *argv[])
     CA_DB *db = NULL;
     BIO *channel;
     time_t *testdateutc = NULL;
+    int r;
```

Comment:
suggestion : name this `rv` as return value, as it is often used in the codebase.